plutus-benchmark/script-contexts/src/PlutusBenchmark/V2/Data/ScriptContexts.hs

```diff
@@ -140,3 +195,82 @@ mkScriptContextEqualityOverheadCode sc =
   in $$(PlutusTx.compile [|| scriptContextEqualityOverhead ||])
     `PlutusTx.unsafeApplyCode` PlutusTx.liftCodeDef sc
     `PlutusTx.unsafeApplyCode` PlutusTx.liftCodeDef d
+
+forwardWithStakeTrick :: BuiltinData -> BuiltinData -> ()
+forwardWithStakeTrick obsScriptCred ctx =
+  case PlutusTx.unsafeFromBuiltinData ctx of
+    ScriptContext { scriptContextTxInfo = TxInfo { txInfoWdrl } } ->
+      let txInfoWdrl' = Map.toBuiltinList txInfoWdrl
+          wdrlAtZero = BI.fst $ BI.head txInfoWdrl'
+          rest = BI.tail txInfoWdrl'
+          wdrlAtOne = BI.fst $ BI.head rest
+      in
+        if PlutusTx.equalsData obsScriptCred wdrlAtZero
+          || PlutusTx.equalsData obsScriptCred wdrlAtOne
+          then ()
+          else
+            if Map.member' obsScriptCred rest
```

Comment:
I don't think you need `member'` here - you can use `PlutusTx.Data.List.elem` (or something very similar).

Even if you do need to expose `member'`, it should be exposed from the List module (since it has nothing to do with Map), and it should have Haddock.